.appveyor.yml

```diff
@@ -0,0 +1,43 @@
+environment:
+  matrix:
+  - julia_version: 0.7
+  - julia_version: 1
+  - julia_version: nightly
+
+platform:
+  - x86 # 32-bit
```

Comment:
We won't work on 32-bit

Reply:
OK, will change.

But is Windows supported at all? If not, perhaps this should be removed completely...

Reply:
No not really, just WSL so in the end only UNIX systems... So just drop this :)

Reply:
@fingolfin Polymake does not work on Windows at all, so no need to support windows.

Reply:
Excellent. I was surprised to see the AppVeyor config in here, happy to remove it instead of updating it :)